app/assets/stylesheets/darkswarm/modals.css.scss

```diff
@@ -72,6 +77,20 @@ dialog .mobile-close-reveal-modal
   }
 }
 
+.reveal-modal .buttons {
+  margin: 0 auto;
+  text-align: center;
+
+  .bottom-close-button {
+    background-color: $orange-500;
+    margin: 0 0 1rem;
+
+    @media only screen and (min-width: 640px) {
```

Comment:
haven't we merged @Matt-Yorkley 's mixin to abstract media breakpoints?

Reply:
Nope, those changes are in the feature branch which is currently blocked by: #5345

Reply:
I cant use it yet, not in master...